sqlite_query.c

```diff
@@ -369,6 +371,16 @@ sqlite_convert_to_pg(Form_pg_attribute att, sqlite3_stmt * stmt, int stmt_colid,
 				}
 				break;
 			}
+		case VARBITOID:
+		case BITOID:
+			{
+				char * buffer = palloc0(SQLITE_FDW_BIT_DATATYPE_BUF_SIZE);			
```

Comment:
Missing type cast: `(char *) palloc0(...`

Reply:
Added, see https://github.com/pgspider/sqlite_fdw/pull/83/commits/ff232874f174af1e29c12d30285bbf3ab512e7c8

Reply:
Thank for fixing. Please help me resolve this thread.